blocks/library/code/index.js

```diff
@@ -31,6 +31,7 @@ registerBlockType( 'core/code', {
 		from: [
 			{
 				type: 'pattern',
+				trigger: 'enter',
```

Comment:
Aside: These patterns are hard to discover. Didn't realize I had to Shift+Enter to trigger the transform.